php-practice_1.php

```diff
@@ -0,0 +1,139 @@
+<?php
+// Q1 変数と文字列
+$name = '三瓶';
+echo '私の名前は' . $name . 'です';
+
+// Q2 四則演算
+$num = 5 * 4;
+echo $num;
+echo $num / 2;
+
+// Q3 日付操作
+echo date("Y/m/d H:i:s");
+echo '現在時刻は、' . date("Y年m月d日 H時i分s秒") . 'です。';
+
+// Q4 条件分岐-1 if文
+$device = 'windows';
+if ($device === 'windows' || $device ==='mac') {
+    echo '使用OSは、'.$device.'です。';
```

Comment:
[nits]
`.`の前後は空白設けてください！
```suggestion
    echo '使用OSは、' . $device . 'です。';
```